tests/service_automation/tests/step_definitions/crud_api_steps/test_organization_api.py

```diff
@@ -705,8 +802,19 @@ def step_update_with_invalid_extension(invalid_scenario: str, api_request_contex
     """Update organization with various invalid extension scenarios."""
     payload = _load_default_payload()
 
-    # Handle role-level validation scenarios
-    if invalid_scenario in ("invalid role extension url", "missing role extension url", "empty role extension url"):
+    # Handle role-level validation scenarios (URL and roleCode structure)
+    role_level_scenarios = (
+        "invalid role extension url",
+        "missing role extension url",
+        "empty role extension url",
+        "missing roleCode extension",
```

Comment:
so many roles now, to be explicit should we make these
```suggestion
        "invalid organisation role extension url",
        "missing organisation role extension url",
        "empty organisation role extension url",
```
?